tests_integration/conftest.py

```diff
@@ -172,6 +173,36 @@ def _temporary_project_directory(
     return _temporary_project_directory
 
 
+@pytest.fixture
+def snow_templating(runner):
```

Comment:
Can you explaing / documment what is the purpose of this fixture? When should it be used?